MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m

```diff
@@ -446,7 +450,38 @@ - (nullable NSURL *)buildTokenRequestURLWithAppSecret:(NSString *)appSecret rele
   return components.URL;
 }
 
-- (void)openURLInEmbeddedSafari:(NSURL *)url fromClass:(Class)clazz {
+- (void)openURLInAuthenticationSessionWith:(NSURL *)url fromClass:(Class)clazz {
```

Comment:
Not on you but I realized seeing `openURLInAuthenticationSessionWith:fromClass:` that we should remove the last `Url` from `openURLInSafariViewControllerWithUrl:fromClass` :)

Reply:
I'll do that!